src/images/projectIcons/rust.svg

```diff
@@ -1 +1,19 @@
14,-0.992 l 5.166,0.181 c 0.521,0.021 1.019,-0.234 1.31,-0.671 0.292,-0.436 0.337,-0.991 0.118,-1.468001 l -2.139,-4.693 c 0.209,-0.379 0.414,-0.759 0.612,-1.145 l 5.09,-0.828 c 0.518,-0.083 0.954,-0.429 1.154,-0.914 0.2,-0.483 0.137,-1.036 -0.17,-1.462 l -3.011,-4.183 c 0.13,-0.413 0.257,-0.828 0.378,-1.246 l 4.827,-1.805 c 0.491,-0.184 0.853,-0.607 0.953,-1.122 0.104,-0.514 -0.068,-1.043 -0.452,-1.4 l -3.776,-3.523 c 0.046,-0.425 0.088,-0.853 0.125,-1.28 l 4.394,-2.72 c 0.445,-0.274 0.716,-0.761 0.716,-1.285 0,-0.524 -0.27,-1.011 -0.715,-1.286 z M 94.207999,107.167 c -1.678,-0.362 -2.745,-2.016 -2.385,-3.699 0.359,-1.681 2.012,-2.751 3.689,-2.389 1.678,0.359 2.747,2.016 2.387,3.696 -0.36,1.68 -2.014,2.751 -3.691,2.392 z m -1.493,-10.098001 c -1.531,-0.328 -3.037,0.646 -3.365,2.18 l -1.56,7.280001 c -4.814,2.185 -10.16,3.399 -15.79,3.399 -5.759,0 -11.221,-1.274 -16.121,-3.552 l -1.559,-7.280001 c -0.328,-1.532 -1.834,-2.508 -3.364,-2.179 l -6.427,1.38 c -1.193,-1.228 -2.303,-2.536 -3.323,-3.917 l 31.272,0 c 0.354,0 0.59,-0.064 0.59,-0.386 l 0,-11.062 c 0,-0.322 -0.236,-0.386 -0.59,-0.386 l -9.146,0 0,-7.012 9.892,0 c 0.903,0 4.828,0.258 6.083,5.275 0.393,1.543 1.256,6.562 1.846,8.169 0.588,1.802 2.982,5.402 5.533,5.402 l 15.583001,0 c 0.177,0 0.366,-0.02 0.565,-0.056 -1.081,1.469 -2.267,2.859 -3.544001,4.158 l -6.575,-1.413 z m -43.252,9.946001 c -1.678,0.362 -3.33,-0.708 -3.691,-2.389 -0.359,-1.684 0.708,-3.337 2.386,-3.699 1.678,-0.359 3.331,0.711 3.691,2.392 0.359,1.68 -0.709,3.336 -2.386,3.696 z m -11.863,-48.105001 c 0.696,1.571 -0.012,3.412 -1.581,4.107 -1.569,0.697 -3.405,-0.012 -4.101,-1.584 -0.696,-1.572 0.012,-3.41 1.581,-4.107 1.57,-0.696 3.405,0.012 4.101,1.584 z m -3.646,8.643 6.695,-2.975 c 1.43,-0.635 2.076,-2.311 1.441,-3.744 l -1.379,-3.118 5.423,0 0,24.444 -10.941,0 c -0.949,-3.336 -1.458,-6.857 -1.458,-10.496 0,-1.389 0.075,-2.762 0.219,-4.111 z m 29.38,-2.374 0,-7.205 12.914,0 c 0.667,0 4.71,0.771 4.71,3.794 0,2.51 -3.101,3.41 -5.651,3.41 l -11.973,0 z m 46.932001,6.485 c 0,0.956 -0.035,1.902 -0.105,2.841 l -3.926,0 c -0.393,0 -0.551,0.258 -0.551,0.643 l 0,1.803 c 0,4.244 -2.393,5.167 -4.49,5.402 -1.997001,0.225 -4.211001,-0.836 -4.484001,-2.058 -1.178,-6.626 -3.141,-8.041 -6.241,-10.486 3.847,-2.443 7.85,-6.047 7.85,-10.871 0,-5.209 -3.571,-8.49 -6.005,-10.099 -3.415,-2.251 -7.196,-2.702 -8.216,-2.702 l -40.603,0 c 5.506,-6.145 12.968,-10.498 21.408,-12.082 l 4.786,5.021 c 1.082,1.133 2.874,1.175 4.006,0.092 l 5.355,-5.122 c 11.221,2.089 20.721,9.074 26.196001,18.657 l -3.666,8.28 c -0.633,1.433 0.013,3.109 1.442,3.744 l 7.058,3.135 c 0.122,1.253 0.186,2.519 0.186,3.802 z m -40.575001,-41.88 c 1.24,-1.188 3.207,-1.141 4.394,0.101 1.185,1.245 1.14,3.214 -0.103,4.401 -1.24,1.188 -3.207,1.142 -4.394,-0.102 -1.184,-1.243 -1.139,-3.212 0.103,-4.4 z m 36.380001,29.277 c 0.695,-1.571 2.531,-2.28 4.1,-1.583 1.569,0.696 2.277,2.536 1.581,4.107 -0.695,1.572 -2.531,2.281 -4.101,1.584 -1.567,-0.696 -2.275,-2.535 -1.58,-4.108 z" /></svg>
```

Comment:
Any idea why this went from a single tag to using both an xml and svg tags?

Reply:
I think that was done for the ease of differentiation.